port_ocean/core/event_listener/kafka.py

```diff
@@ -122,7 +122,9 @@ async def _handle_message(self, raw_msg: Message) -> None:
 
         if "change.log" in topic and message is not None:
             try:
+                await self._before_resync()
                 await self.events["on_resync"](message)
+                await self._after_resync()
```

Comment:
I think we can add the `resync` method to the `baseEventListener` that will contain this code that way you won't need to repeat it for each event listener, as well as making sure that every new event listener added will get it out of the box

Reply:
Is it a blocker?